src/aat/resources/environment.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.scheduler.url=jdbc:postgresql://localhost:5431/scheduler?ssl
+spring.datasource.scheduler.url=jdbc:postgresql://localhost:5431/scheduler
```

Comment:
Does it really matter?

Reply:
no it doesn't

Reply:
I guess no point merging this? Shame we need to add these props for AAT test runner.